### PR TITLE
enable prometheus for scale-out 

### PR DESCRIPTION
**Pull request by pdgetrf**
### Changes 
add changes from the following commits to enable prometheus in scale-out cluster

https://github.com/futurewei-cloud/arktos-perftest/commit/e5f6915b1603e29fc76f6ec6af3739573165ddd8
https://github.com/futurewei-cloud/arktos-perftest/commit/c808cf317b2a0d6b16a672f1cfe81709c3c54a2b

### CODE REVIEW

**Comment by pdgetrf** [comment]
@sonyafenge other than the changes here, is there anything else we need to do to enable prometheus when starting the cluster?

**Comment by sonyafenge** [comment]
> @sonyafenge other than the changes here, is there anything else we need to do to enable prometheus when starting the cluster? for example, env variables like PROMETHEUS_SCRAPE_ETCD=true ENABLE_PROMETHEUS_SERVER=true

no, we don't need any env variables because this is a temporally solution without switch variables.  ENABLE_PROMETHEUS_SERVER=true is the one supported by community and arktos code, I have a quick check and cannot get this one access outside of cluster.

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -156,6 +156,7 @@ function main() {
   reset-motd
   prepare-mounter-rootfs
   modprobe configs
+  start-prometheus &  #####start prometheus
```
Comment (sonyafenge) [inline cluster/gce/gci/configure-helper.sh:159]: this one only for main master, if you have partitioned server, you need double check if your partitioned server also has this started.

**Review by sonyafenge — APPROVED** [review]
LGTM

**Comment by XiaoningDing** [comment]
Remove the "do not merge" label as Qian confirmed the verification is done.

**Pull request merged**